resources/schemas/dbscripts/postgresql/targetedms-19.19-19.20.sql

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
Let's not go all the way to 19.20, since there's a late-breaking script that we want to add.

Luckily we can use an extra decimal place, so please make this 19.19-19.191 instead.

Reply:
done